action.yaml

```diff
@@ -40,6 +40,12 @@ inputs:
       Required if running in migration mode.
     required: false
     type: string
+  base_branch:
```

Comment:
Perhaps we should default to `main`. Thoughts?

Reply:
It is currently optional and results in using the default branch, which could be `main` depending on how the repo is setup. If we set it to `main` and a repo has a different name for the default branch, they the developer needs to manually set it. Does that make sense?

Reply:
Perfect sense. Thx for clarifying